scene/gui/code_edit.cpp

```diff
@@ -1764,6 +1823,95 @@ TypedArray<int> CodeEdit::get_folded_lines() const {
 	return folded_lines;
 }
 
+/* Code region */
+void CodeEdit::create_code_region() {
+	// Abort if there is no selected text
+	if (!has_selection()) {
+		return;
+	}
+	// Check that region tag find a comment delimiter and is valid
+	if (code_region_start_string.is_empty()) {
+		WARN_PRINT_ONCE("Cannot create code region without any one line comment delimiters");
+		return;
+	}
+	begin_complex_operation();
+	// Merge selections if selection starts on the same line the previous one ends
```

Comment:
```suggestion
	// Merge selections if selection starts on the same line the previous one ends.
```

Reply:
You have missed this and several others, please pay attention to the hidden review comments

Reply:
Sorry. I've check all // comments and add missing points.
Some were forgiven by mistake but for some others were on purpose because I thought that if it was not a sentence with a conjugated verb, it doesn't need final period. Now periods added everywhere.